### PR TITLE
Ingestion visualization

### PR DESCRIPTION
**Pull request by therealslimjp**
POC. If you got another ideas on what i should add, feel free to leave a comment

### CODE REVIEW

**Comment by rakeshJn** [comment]
Thanks, will take a look soon.

File: lake_viewer.py [hunk 1]
```diff
@@ -10,8 +10,9 @@
 from streamlit.components.v1 import html
 import json
 import dotenv
+from table_tabs.ingestion_statistics import show_ingestion_patterns
 dotenv.load_dotenv(dotenv.find_dotenv(usecwd=True)) #Use current working directory to load .env file
-
+print(os.environ)
```
Comment (jurossiar) [inline lake_viewer.py:15]: Please, remove this print

Comment (therealslimjp) [reply lake_viewer.py:15]: Ofc. Forgot to take it out. Debugging…

**Comment by jurossiar** [comment]
Seems Granularity is not working as expected. I think should be grouped by hour, day, week in the X axis
E.g.:
image
image
image
image

**Comment by rakeshJn** [comment]
Hi @therealslimjp , sorry it took a while with all the re-architecture being done. In the new set up, I have tried to capture what you were trying to achieve through this PR. Please check: https://github.com/IBM/lakevision/pull/45
I have merged it to main so you can try. It does open a lot of other possibilities for insights I think.
Please try out the new one, it is much cleaner as we can separate out python side as pure rest api and front end side in Svelte/JS.
I will close this PR. Also there are several issues open if you want to pick any of those..... Thanks.

**Pull request closed without merging**